snap/info_snap_yaml.go

```diff
@@ -53,9 +60,23 @@ type slotYaml struct {
 }
 
 type appYaml struct {
-	Command   string   `yaml:"command"`
-	SlotNames []string `yaml:"slots,omitempty"`
-	PlugNames []string `yaml:"plugs,omitempty"`
+	Command string `yaml:"command"`
+
+	Daemon string `yaml:"daemon"`
+
+	Stop        string          `yaml:"stop-command,omitempty"`
+	PostStop    string          `yaml:"post-stop-command,omitempty"`
+	StopTimeout timeout.Timeout `yaml:"stop-timeout,omitempty"`
+
+	RestartCond systemd.RestartCondition `yaml:"restart-condition,omitempty"`
+	SlotNames   []string                 `yaml:"slots,omitempty"`
+	PlugNames   []string                 `yaml:"plugs,omitempty"`
+
+	BusName string `yaml:"bus-name,omitempty"`
```

Comment:
This field in 15.04 served two purposes: 1) it was used in the generation of 15.04 simplistic dbus bus policy and 2) if it was specified then 'Type=dbus' would be used in the systemd unit.

'1' is handled completely by interfaces. I think we decided the other day that it is up to the snap developer to choose 'daemon: dbus' (which is supported today) if he/she wants 'Type=dbus' in the systemd unit. As such, I think BusName can and should be removed from the yaml.